pyalgostrategypool/aaroon_crossover_v2.py

```diff
@@ -0,0 +1,152 @@
+import talib
```

Comment:
remove the following from all strategies:

- all imports
- all comments
- all docstrings

comments and docstrings should be captured somewhere in the generic or specific doc

do this for all strategies

Reply:
done